src/applications/facility-locator/components/search-results/LocationPhoneLink.jsx

```diff
@@ -60,21 +77,51 @@ const LocationPhoneLink = ({ location, from, query }) => {
             If you don’t have a referral, contact your local VA medical center.
           </p>
         )}
+      </dd>
+    ) : (
+      <div>
+        {renderPhoneNumber(
+          isCCProvider ? 'If you have a referral' : null,
+          isCCProvider ? 'Call this facility at' : null,
```

Comment:
Since there is already a lot of logic inside of `renderPhoneNumber` that checks `isCCProvider`, it could make sense to just omit these two arguments from `renderPhoneNumber` and have that helper function derive the values itself. `renderPhoneNumber ` is used in so many different ways with that argument list though it might be easier to just separate it out entirely into `ProviderPhone` and `FacilityPhone` with some prop types to make the logic more straightforward.

Also, since `renderPhoneNumber` returns JSX, it's really a JSX component itself, so it could be renamed to `PhoneContent` or something.